src/app/dashboard/events/[id]/forms/(steps)/general-info.tsx

```diff
@@ -243,7 +249,7 @@ function GeneralInfoForm({ goToNextStep }: { goToNextStep: () => void }) {
                     <FormControl>
                       <Textarea
                         {...field}
-                        className="h-5/6 resize-none"
+                        className="h-55 resize-none"
```

Comment:
```suggestion
                        className="max-h-55 min-h-14"
```
Osobiście bym to ustawił na maksymalną wysokość, ustawił minimalną i pozwolił użytkownikowi na resizoweanie